### PR TITLE
Add global warning and info page for ShARC decommissioning.

### PR DESCRIPTION
**Pull request by jkwmoore**
Merge pending on approval / a comms email being developed.

Comms email should include the ' information on issues that impede migration from ShARC to Stanage' Google form which is being omitted from public viewing here.

### CODE REVIEW

**Comment by desmfryan** [comment]
Looks good to me James

**Review by nikishe — APPROVED** [review]
LGTM

**Review by Carldkennedy — APPROVED** [review]
LGTM

File: sharc/sharc-decommissioning.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+.. _sharc_decommissioning:
+
+ShARC decommissioning
+=====================
+
+**ShARC will be decommissioned on 30th November 2023, after which time users will no longer be able to access that cluster and any jobs running or 
+queueing at that time will be cancelled.**
+
+Once ShARC has been decommissioned and all data securely deleted the system is to be donated to the `Taras Shevchenko National University of Kyiv <http://www.univ.kiev.ua/en/>`_.
```
Comment (jkwmoore) [inline sharc/sharc-decommissioning.rst:9]: This is incorrect and needs amendment to: 

```suggestion
Once ShARC has been decommissioned and all data securely deleted the system is to be donated to the `Igor Sikorsky Kyiv Polytechnic Institute <https://kpi.ua/en>`_ in Kyiv, Ukraine.
```

File: sharc/sharc-decommissioning.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Equivalent shared application installation areas will be made available on the Stanage cluster prior to the ShARC decommissioning.
+
+----
+
+Stanage is our recommended target for migration 
+"""""""""""""""""""""""""""""""""""""""""""""""
+
+Stanage is our newest HPC cluster and is the logical successor to ShARC; it is well suited to running larger parallel jobs spanning multiple nodes and benefits from a high-performance network interconnect between nodes.  
+
+As Bessemer only permits single node execution and has approximately 18 months of life left, we recommend that ShARC users preferentially migrate workloads to Stanage instead.
```
Comment (willfurnass) [inline sharc/sharc-decommissioning.rst:55]:
```suggestion
As Bessemer only permits single node execution and is several years older than Stanage, we recommend that ShARC users preferentially migrate workloads to Stanage instead.
```

**Review by willfurnass — CHANGES_REQUESTED** [review]

**Review by willfurnass — APPROVED** [review]

**Pull request merged**